plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java

```diff
@@ -309,4 +319,34 @@ public static List<Column> toHiveColumns(List<NestedField> columns)
                         Map.of()))
                 .collect(toImmutableList());
     }
+
+    private void deleteRemovedMetadataFiles(TableMetadata base, TableMetadata metadata)
```

Comment:
Why did you copy the code from `CatalogUtil.deleteRemovedMetadataFiles` instead of calling the method?

Reply:
Sorry, I didn't know this method was already public in iceberg. ,I updated the code. thank you.